attributes/monitors.rb

```diff
@@ -61,12 +61,12 @@
   chk['details']['timeout'] = 60
   # Platformstack wants the alarm hash
   # Can override this in the wrapper/role/env
-  chk['alarm'] = {}
+  chk['alarm'] = false
```

Comment:
Indeed, it's a shame because everything else would be compatible. I was thinking maybe release this as a Major change, then do an other PR later which will really stop to support any platformstack logic/attribute. @martinb3 Thoughts ?